<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetsImportView</class>
 <widget class="QWidget" name="WidgetsImportView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lbLocale">
       <property name="text">
        <string>Locale:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbLocale"/>
     </item>
     <item>
      <widget class="QLabel" name="lbGroupSeparator">
       <property name="text">
        <string>Grouping symbol:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbGroup"/>
     </item>
     <item>
      <widget class="QLabel" name="lbDecimal">
       <property name="text">
        <string>Decimal separator:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbDecimal"/>
     </item>
     <item>
      <widget class="QLabel" name="lbDateTime">
       <property name="text">
        <string>Date Time Format:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leDateTime">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maxLength">
        <number>25</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbDateTimeSample">
       <property name="text">
        <string>Example:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbDatePreview">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="ShowPreview">
       <property name="text">
        <string>Preview</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QTableView" name="SourceTable">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
     </widget>
     <widget class="QTableView" name="PreviewTable">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
